R/chrome.R

```diff
@@ -118,8 +121,19 @@ chrome_print = function(
 
   box_model = match.arg(box_model)
 
+  pr = NULL
+  res_fun = NULL
+  if (async <- async && xfun::loadable('promises')) {
```

Comment:
I'd just do this

```suggestion
  if (async) {
```

i.e. signal an error (from `promises::promise`) if the **promises** package is not installed. Of course, we need to document it.

Reply:
@cderv Since you marked this as resolved, did you mean you didn't want this change?

Reply:
sorry to be unclear, my comment was about rejecting the promise.

Reply:
No worries!